### PR TITLE
Desktop version- pull request 

### PR DESCRIPTION
**Pull request by AlaaShaheen2**
added a media query for css files to convert to desktop version
added a grid and flexbox display 
changed from mobile version to desktop version

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -418,3 +483,155 @@ header > .logo {
...
+  }
+
+  .phar {
+    margin: 50px 0 20px 0;
+  }
+
+  .navbar {
+    display: flex;
+    flex-direction: row;
+  }
```
Comment (vechicin) [inline styles.css:517]: The navbar font doesn't match that of the Figma template. Please fix it.

Screen Shot 2022-07-15 at 11 20 58

Screen Shot 2022-07-15 at 11 21 25

File: styles.css [hunk 2]
```diff
@@ -418,3 +483,155 @@ header > .logo {
...
 }
+
+@media (min-width: 768px) {
+  .homepage {
+    padding: 50px 30px 100px 30px;
+    height: 600px;
+    margin: 0 auto;
+    background: white;
+    border-radius: 0 0 0 50px;
+  }
```
Comment (vechicin) [inline styles.css:494]: This section looks very dissimilar from the Figma design. You should fix the margin with the navbar, the section's height, the background image positioning, and the width of the title container (I'm Alaa, Glad to see you!)

Screen Shot 2022-07-15 at 11 24 09

Screen Shot 2022-07-15 at 11 24 53

File: styles.css [hunk 3]
```diff
@@ -201,12 +269,11 @@ header > .logo {
...
-  align-items: center;
-  margin: 5px 0;
+  flex-direction: row;
+  list-style: none;
+  margin: 0 0;
   padding: 0;
-  margin-top: 0;
 }
 
 .lang-styles {
```
Comment (vechicin) [inline styles.css:279]: Please use the right font for the list of technologies.

Screen Shot 2022-07-15 at 11 28 33

Screen Shot 2022-07-15 at 11 28 47

File: styles.css [hunk 4]
```diff
@@ -418,3 +483,155 @@ header > .logo {
...
+  .works4 {
+    display: flex;
+    flex-direction: row-reverse;
+  }
+
+  .about-me {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 10px;
+  }
```
Comment (vechicin) [inline styles.css:565]: This section looks very dissimilar to the Figma template. Please fix the margin with the section above, the width of the content, the margin between the section's header, subheader, social media icons, and button, the space between the social media icons, and the button's alignment as well as the border radius.

Screen Shot 2022-07-15 at 11 32 01

Screen Shot 2022-07-15 at 11 32 17

File: styles.css [hunk 5]
```diff
@@ -418,3 +483,155 @@ header > .logo {
...
+    padding: 40px 0;
+    background-color: #6070ff;
+    border-radius: 90px 0 0 0;
+  }
+
+  .contact-form {
+    width: 500px;
+    margin-bottom: 20px;
+    background: none;
+  }
```
Comment (vechicin) [inline styles.css:618]: The form looks a little dissimilar to the Figma template. Please fix the margin between the paragraph and the form's input fields.

Screen Shot 2022-07-15 at 11 37 06

Screen Shot 2022-07-15 at 11 37 23

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @AlaaShaheen2 ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Good job overall following the template design.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 3 above)
Comment (AlaaShaheen2) [reply styles.css:279]: @vechicin  done

File: styles.css (hunk 5 above)
Comment (AlaaShaheen2) [reply styles.css:618]: @vechicin done

File: styles.css (hunk 4 above)
Comment (AlaaShaheen2) [reply styles.css:565]: @vechicin done

**Review by ishimwezachee — APPROVED** [review]
Hi @AlaaShaheen2 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters are passing 👍 
PR is well described 👍 
Project is running successfully 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AlaaShaheen2** [comment]
@ishimwezachee thank you :)

**Pull request merged**